### PR TITLE
Adds a CLI to Rooibos!

### PR DESCRIPTION
**Pull request by markwpearce**
Very simple way to run tests from the command line!

`npx rooibos-roku --project=<path_to_bsconfig.json> --host=<host> --password=<password>`

This will build the tests, deploy them to the given roku device, and pipe output to the console, closing the connection when done (or on some error).

I think this will be very useful for CI processes.

### CODE REVIEW

**Review by TwitchBronBron — COMMENTED** [review]
Is there any chance this could be targeted against the v5 branch instead? I don't think there's anything v6 specific in here, and v5 users could benefit from this much sooner (and we can merge into v6 as soon as this PR lands.

**Comment by markwpearce** [comment]
Yeah, I can.
This does use the 4.0.0-alpha version of `roku-deploy` ... is that still okay to use, or should I change it to use the latest v3?

**Comment by TwitchBronBron** [comment]
If it's not too much of a lift to switch to v3 roku-deploy, that would be ideal. but don't waste too much time on it, v4 is pretty stable...

File: bsc-plugin/src/cli.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    .option('project', { type: 'string', description: 'Path to a bsconfig.json project file.' })
+    .option('host', { type: 'string', description: 'Host of the Roku device to connect to. Overrides value in bsconfig file.' })
+    .option('password', { type: 'string', description: 'Password of the Roku device to connect to. Overrides value in bsconfig file.' })
+    .option('log-level', { type: 'string', defaultDescription: '"log"', description: 'The log level. Value can be "error", "warn", "log", "info", "debug".' })
+    .check((argv) => {
+        if (!argv.host) {
+            return new Error('You must provide a host. (--host)');
+        }
+        if (!argv.password) {
+            return new Error('You must provide a password. (--host)');
```
Comment (TwitchBronBron) [inline bsc-plugin/src/cli.ts]:
```suggestion
            return new Error('You must provide a password. (--password)');
```

**Review by TwitchBronBron — APPROVED** [review]

**Pull request merged**